EIPS/eip-5164.md

```diff
@@ -39,28 +39,71 @@ The `CrossChainRelayer` lives on the chain from which messages are sent. The Rel
 
 #### Methods
 
+**relayCall**
+
+Will relay a single `Call` struct as a batch of one to be executed by any `CrossChainExecutor`(s) that executes the calls from this Relayer.
+
+`CrossChainRelayer`s MUST emit the `RelayedCalls` event when a batch of calls is relayed.
+
+`CrossChainRelayer`s MAY ignore the `toChainId` when there is one or more predetermined destination chain(s).
```

Comment:
Users may want to relay a batch of calls to one or more chains, so I'm wondering if `toChainId` should be a `uint256` array?